daemon/mgr/network.go

```diff
@@ -695,6 +695,13 @@ func buildSandboxOptions(config network.Config, endpoint *types.Endpoint) ([]lib
 
 	// TODO: secondary ip address
 	// TODO: parse extra hosts
+	for _, extraHost := range endpoint.ExtraHosts {
+		// allow IPv6 addresses in extra hosts; only split on first ":"
+		parts := strings.SplitN(extraHost, ":", 2)
+		fmt.Printf("\nparts[0]: %s, \n parts[1]: %s\n", parts[0], parts[1])
```

Comment:
change to `logrus.Debugf` here

Reply:
This is an extra statement. I forget to remove it.